llvm/include/llvm/ADT/DenseMapInfo.h

```diff
@@ -142,7 +142,7 @@ template<> struct DenseMapInfo<unsigned long> {
     if constexpr (sizeof(Val) == 4)
       return DenseMapInfo<unsigned>::getHashValue(Val);
     else
-      return detail::combineHashValue(Val >> 32, Val);
+      return detail::combineHashValue(Val >> (4 * sizeof(Val)), Val);
```

Comment:
```suggestion
      return densemap::detail::mix(Val);
```
to avoid the issue entirely?

Reply:
Sure, I'll update it in a sec (sorry don't know this code very well at all)

Reply:
Could you please also update this for the unsigned long long case, just so the code is the same?